test/.eslintrc.json

```diff
@@ -1,13 +1,12 @@
 {
+	"root": true,
+
+	"extends": "../.eslintrc-browser.json",
+
 	"env": {
 		"browser": true
```

Comment:
So we always want to reference global vars from `window` object? Yeah I recall we talked about this before, sounds good to me then – a default disabled `browser` option and additional comment